.github/workflows/publish.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
+        with:
+          python-version: 3.11
+      - run: |
+          pip install build
+          python -m build
+      - uses: actions/upload-artifact@v3
+        with:
+          path: ./dist
+
+  pypi-publish:
+    needs: ['build']
+    environment: 'publish'
+
+    name: upload release to PyPI
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+    steps:
+      - uses: actions/download-artifact@v3
+      - name: Publish package distributions to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          packages_dir: artifact/
```

Comment:
Is this the correct path? I would have assumed that the artifact would recreate the `dist` folder which is also the default value for this argument.

Reply:
Yes, this is the correct path. You can see it in the job where I replaced the PyPI upload with a `ls -l` to verify this: https://github.com/conda/conda-pack/actions/runs/5822896292/job/15788578370#step:3:6